templates/pcab/image.tpl.html

```diff
@@ -1,3 +1,3 @@
-<div class="images"><img src='<%= file.url %>' alt='<%= description %>'>
+<div class="images"><img src='./img/<%= file.name %>' alt='<%= description %>'>
```

Comment:
Indentation in this file should look the same as in `templates/default/image.tpl.html`